datumaro/plugins/vgg_face2_format.py

```diff
@@ -64,9 +91,18 @@ def _load_items(self, path):
             for row in bboxes_table:
                 if len([p for p in row if row[p] == '']) == 0 and len(row) == 5:
                     item_id = row['NAME_ID']
+                    label = osp.dirname(item_id)
+                    if 0 < len(label):
+                        item_id = osp.basename(item_id)
+
                     annotations = items[item_id].annotations
-                    annotations.append(Bbox(int(row['X']), int(row['Y']),
-                        int(row['W']), int(row['H'])))
+                    label = self._categories[AnnotationType.label].find(label)[0]
+                    if label != None:
```

Comment:
You can just pass `None`, when there is no label. I think, we need to replace existing `Label` with `Bbox`. Or we need to only generate `Label`s when there is no other annotations.